controllers/front/view.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/**
+ * 2007-2020 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+use PrestaShop\Module\BlockWishlist\WishList;
+
+class BlockWishListViewModuleFrontController extends ModuleFrontController
+{
+    public function __construct()
+    {
+        parent::__construct();
+        $this->context = Context::getContext();
+    }
+
+    public function initContent()
+    {
+        parent::initContent();
+        $token = Tools::getValue('token');
+        $module = new BlockWishList();
+
+        if (false === empty($token)) {
+            $wishlist = WishList::getByToken($token);
+
+            WishList::refreshWishList($wishlist['id_wishlist']);
+            $products = WishList::getProductByIdCustomer(
+                (int) $wishlist['id_wishlist'],
+                (int) $wishlist['id_customer'],
+                $this->context->language->id,
+                null,
+                true
+            );
+
+            $nb_products = count($products);
+            $priority_names = [0 => $module->l('High'), 1 => $module->l('Medium'), 2 => $module->l('Low')];
+
+            for ($i = 0; $i < $nb_products; ++$i) {
+                $obj = new Product((int) $products[$i]['id_product'], true, $this->context->language->id);
```

Comment:
```suggestion
                $product = new Product((int) $products[$i]['id_product'], true, $this->context->language->id);
```
Use meaningful variable names, as they help you write readable code